src/openvic-simulation/pop/Pop.cpp

```diff
@@ -125,6 +127,18 @@ void Pop::set_location(ProvinceInstance const& new_location) {
 	}
 }
 
+void Pop::update_gamestate(DefineManager const& define_manager, fixed_point_t const& min_pop_size_for_regiment_multiplier) {
+	if (type.get_can_be_recruited()) {
```

Comment:
There should be a check if the culture is recruitable. Unit types can be restricted to primary or accepted cultures.
If pop is not of an accepted culture and all unit types require that, then the pop can't be recruited

Reply:
Good catch, I've added a system for keeping track of which cultures are recruitable based on currently unlocked regiment types.